ipam/pool.go

```diff
@@ -379,14 +386,34 @@ func (as *addressSpace) releasePool(poolId string) error {
 //
 // AddressPool
 //
-
-// Returns if an address pool is currently in use.
+// Returns AddressPoolInfo with Ip address statistics
 func (ap *addressPool) getInfo() *AddressPoolInfo {
+
+	var available, capacity int = 0, 0
+	var unhealthyAddr []string
+
+	for _, ar := range ap.Addresses {
+		if !ar.InUse {
+			available += 1
+		}
+		if ar.Unhealthy {
+			addr := net.IPNet{
+				IP:   ar.Addr,
+				Mask: ap.Subnet.Mask,
+			}
+			unhealthyAddr = append(unhealthyAddr, addr.String())
+		}
+		capacity += 1
+	}
+
 	info := &AddressPoolInfo{
-		Subnet:     ap.Subnet,
-		Gateway:    ap.Gateway,
-		DnsServers: []net.IP{dnsHostProxyAddress},
-		IsIPv6:     ap.IsIPv6,
+		Subnet:        ap.Subnet,
+		Gateway:       ap.Gateway,
+		DnsServers:    []net.IP{dnsHostProxyAddress},
+		IsIPv6:        ap.IsIPv6,
+		Capacity:      capacity,
```

Comment:
Is there a need for a separate capacity variable? Capacity is simply len(ap.Addresses).